simple_app/utils.py

```diff
@@ -204,53 +316,96 @@ def __init__(self, *args, **kwargs):
         return
 
     def validate_sqlfield(self, field):
+        """
+        Validating SQL queries before they can be submitted
+        Parameters
+        ----------
+        field
+            The data within the query form
+        """
+        forbidden = ('update', 'drop', 'truncate', 'grant', 'commit', 'create', 'replace', 'alter', 'insert')
```

Comment:
Should add `delete`, though the built in sqlite/sqlalchemy checks caught this one and didn't let me delete anything